### ISSUE TITLE
can't reach the performace as you say ,why?

### ISSUE THREAD

**Issue by xiaojuan123** [kweisamx/VDSR#3]
I have traind 20 layers ,but my Generated HR images are not the same quality as Original one.I want ask help for you?what mistake I have made??thanks

**Comment by kweisamx** [comment]
What number iter did you train? maybe can purposely train few image, make overfitting and observe the image .

**Comment by xiaojuan123** [comment]
yes,you remind  me ,thank you very much,I have 60000 iterations,it's will be overfit，I will try again
